Backpack/Calendar/Classes/BPKCalendarCellSpacing.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKCalendarCellSpacing : NSObject
```

Comment:
My one concern here is the extra cost of Objective-C method dispatch, but that's probably a premature optimisation. Can we add a commen there stating something to the effect of "If we encounter performance issues with the calendar we should switch these properties to C functions which the compiler could inline"?